Failed to activate the hidpi package

[Enter steps to reproduce below:]

1. Upgraded from Atom 1.7.3 to Atom 1.7.4 using Fedora Copr package.
2. When opening Atom for the first time after that, I received the error.

**Atom Version**: 1.7.4
**System**: "Fedora release
**Thrown From**: [hidpi](https://github.com/mcanders/atom-hidpi) package, v0.11.0


### Stack Trace

Failed to activate the hidpi package

```
At Cannot find module 'web-frame'

Error: Cannot find module 'web-frame'
    at Module._resolveFilename (module.js:438:15)
    at Module._resolveFilename (/usr/lib64/electron/resources/electron.asar/common/reset-search-paths.js:47:12)
    at Function.Module._resolveFilename (/usr/lib64/atom/src/module-cache.js:389:52)
    at Function.Module._load (module.js:386:25)
    at Object.<anonymous> (/home/razican/.atom/packages/hidpi/lib/hidpi.coffee:2:12)
    at Object.<anonymous> (/home/razican/.atom/packages/hidpi/lib/hidpi.coffee:1:1)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Package.module.exports.Package.requireMainModule (/usr/lib64/atom/src/package.js:715:27)
    at Package.module.exports.Package.activateNow (/usr/lib64/atom/src/package.js:173:16)
    at /usr/lib64/atom/src/package.js:156:32
    at Package.module.exports.Package.measure (/usr/lib64/atom/src/package.js:92:15)
    at /usr/lib64/atom/src/package.js:149:26
    at Package.module.exports.Package.activate (/usr/lib64/atom/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/lib64/atom/src/package-manager.js:530:21)
    at /usr/lib64/atom/src/package-manager.js:511:29
    at Config.module.exports.Config.transactAsync (/usr/lib64/atom/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/usr/lib64/atom/src/package-manager.js:506:19)
    at PackageManager.module.exports.PackageManager.activate (/usr/lib64/atom/src/package-manager.js:489:46)
    at /usr/lib64/atom/src/atom-environment.js:754:28
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "projectHome": "/home/razican/workspace"
  },
  "hidpi": {}
}
```

### Installed Packages

```coffee
# User
hidpi, v0.11.0
language-rust, v0.4.6
language-x86asm, v0.8.0
linter, v1.11.4
linter-rust, v0.4.6

# Dev
No dev packages
```

--- a/lib/hidpi.coffee
+++ b/lib/hidpi.coffee
@@ -1,5 +1,5 @@
 {CompositeDisposable} = require 'atom'
-WebFrame = require 'web-frame'
+WebFrame = require('electron').webFrame
 class Hidpi
   subscriptions: null
   currentScaleFactor: 1.0

